docs/modules/operation/pages/performance-data-collection/snmp-property-extenders/value-mapping.adoc

```diff
@@ -0,0 +1,37 @@
+= Value Mapping Property extender
+The Value Mapping property extender provides a mechanism that works like a lookup table for values of a local MIB table column.
+
+== When to use the Value Mapping Property extender
+Use the Value Mapping property extender to map an enumerated set of values (e.g. strings) to a corresponding set of values.
+This allows to persist changes of string enumeration values as integers over time as time-series.
```

Comment:
```suggestion
This allows OpenNMS to persist changes of string enumeration values as integers over time as time-series.
```
Object was missing. Not sure if OpenNMS is the right word here.

Reply:
Yes. It is.